SPECS/libidn2/libidn2.spec

```diff
@@ -33,30 +37,29 @@ developing applications that use libidn2.
 
 %package -n idn2
 Summary:          IDNA2008 internationalized domain names conversion tool
-License:          GPLv3+
+License:          GPL-3.0-or-later
 Requires:         %{name}%{?_isa} = %{version}-%{release}
 %if 0%{?rhel} && 0%{?rhel} <= 7
```

Comment:
since we're not rhel.  I think you can purge the macro here.  If I'm reading the macro correctly, it seems like we don't even process lines 43 or 44 in Azure Linux.

Reply:
ack